app/controllers/application_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ApplicationController < ActionController::Base
+ skip_before_action :verify_authenticity_token
+ protect_from_forgery with: :null_session
+
```

Comment:
For reference: use of doorkeeper
https://github.com/developerJai/rails-chat-action-cable/blob/0bf0543ffce7f04d03411ee7f5498f4caedea138/app/controllers/api/v1/application_controller.rb